src/sentry/utils/raven.py

```diff
@@ -34,16 +35,26 @@ def is_enabled(self):
             return False
         return settings.SENTRY_PROJECT is not None
 
-    def capture(self, *args, **kwargs):
```

Comment:
Because of `build_msg`? https://github.com/getsentry/raven-python/blob/master/raven/base.py#L556-L558

I can refactor this a bit to avoid.